packages/react-instantsearch-hooks-dom/src/ui/Hits.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
+import type { JSXElementConstructor } from 'react';
+
+export type HitsProps<THit> = React.ComponentProps<'div'> & {
```

Comment:
There's the following note in the React types for `ComponentProps`:

```txt
NOTE: prefer ComponentPropsWithRef, if the ref is forwarded,
or ComponentPropsWithoutRef when refs are not supported.
```

We don't forward `ref`s in this component, but the user could technically pass one (although it wouldn't work and generate a warning in the console). Could we hint them by using `ComponentPropsWithoutRef`? Also, what's the pro of using `React.ComponentProps` vs. `React.HTMLAttributes<HTMLDivElement>`? I usually go for the latter, interested in the choice.

Reply:
I just copied this from somewhere else, `React.HTMLAttributes<HTMLDivElement>` seems easiest indeed

Reply:
```suggestion
export type HitsProps<THit> = React.HTMLAttributes<HTMLDivElement> & {
```